instance/tests/integration/factories/instance.py

```diff
@@ -79,8 +79,8 @@ def create(cls, *args, **kwargs):  # pylint: disable=arguments-differ
     # or a release candidate tag will work.  We point both the edx-platform and the configuration
     # versions to the branch "integration" in our own forks.  These branches are based on the
     # corresponding openedx_release versions from upstream, but can contain custom modifications.
-    openedx_release = 'open-release/koa.3'
+    openedx_release = 'open-release/lilac.2'
     configuration_source_repo_url = 'https://github.com/open-craft/configuration.git'
-    configuration_version = 'integration-koa.3'
+    configuration_version = 'opencraft-release/lilac.2'
```

Comment:
This will have to be changed to `integration-lilac.2`, once https://tasks.opencraft.com/browse/BB-4996 is ready.

Reply:
@mtyaka Would you consider this an external blocker for this ticket? Or can this be merged and a follow-up PR done?

Reply:
I'm not sure how the code in the integration branch differs from the standard release branch. If integration tests work with the standard branch, then I guess we don't need to wait for the integration branch to be ready. I guess we'll see if tests on this PR work :)